test/retcodes_test.py

```diff
@@ -170,3 +171,68 @@ def new_func(*args, **kwargs):
         with mock.patch('luigi.scheduler.Scheduler.add_task', new_func):
             self.run_and_expect('RequiringTask', 0)
             self.run_and_expect('RequiringTask --retcode-not-run 5', 5)
+
+class RetcodesWithTaskRetryTest(LuigiTestCase):
```

Comment:
The same thing here. If you compare this test with the others, it's much much harder to read and understand.